blog/2018-10-04-debezium-0-9-0-alpha2-released.adoc

```diff
@@ -0,0 +1,124 @@
+you might alternatively represent them as a string-ified JSON by setting the new `hstore.handling.mode` connector option to `json`.
+In this case, you'd see HSTORE columns represented as values in change messages like so: `{ "key1" : "val1", "key2" : "val2" }`.
+
+== Field filtering and renaming for MongoDB
+
+Unlike the connectors for MySQL and Postgres, the Debezium MongoDB connector so far didn't allow to exclude single fields of captured collections from CDC messages.
+Also renaming them wasn't supported e.g. by means of Kafka's `ReplaceField` SMT.
+The reason being that MongoDB doesn't mandate a fixed schema for the documents of a given collection,
+and documents therefore are represented in change messages using a single string-ified JSON field.
+
+Thanks to the fantastic work of community member Andrey Pustovetov,
+this finally has changed, i.e. you can remove given fields (https://issues.jboss.org/browse/DBZ-633[DBZ-633]) now from the CDC messages of given collections or have them renamed (https://issues.jboss.org/browse/DBZ-881[DBZ-881]).
+Please refer to the description of the new connector options `field.blacklist` and `field.renames` in the link:/docs/connectors/mongodb/[MongoDB connector documentation] to learn more.
+
+== Extended source info
+
+Another contribution by Andrey is the new optional `connector` field within the source info block of CDC messages
+(https://issues.jboss.org/browse/DBZ-918[DBZ-918]).
+This tells the type of source connector that produced the messages ("mysql", "postgres" etc.),
+which can come in handy in cases where specific semantics need to be applied on the consumer side depending on the type of source database.
+
+== Bug fixes and version upgrades
+
+The new release contains a good number of bug fixes and other smaller improvements.
+Amongst them are
+
+* correct handling of invalid temporal default values with MySQL (https://issues.jboss.org/browse/DBZ-927[DBZ-927]),
+* support for table/collection names with special characters for MySQL (https://issues.jboss.org/browse/DBZ-878[DBZ-878]) and MongoDB (https://issues.jboss.org/browse/DBZ-865[DBZ-865]) and
+* fixed handling of blacklisted tables with the new Antlr-based DDL parser (https://issues.jboss.org/browse/DBZ-872[DBZ-872]).
+
+Community member Ian Axelrod provided a fix for a potential performance issue,
+where changes to tables with TOAST columns in Postgres would cause repeated updates to the connector's internal schema metadata,
+which can be a costly operation (https://issues.jboss.org/browse/DBZ-911[DBZ-911]).
+Please refer to the link:/docs/connectors/postgres/[Postgres connector documentation] for details on the new `schema.refresh.mode` option,
+which deals with this issue.
+
+In terms of version upgrades with migrated to the latest releases of the MySQL (https://issues.jboss.org/browse/DBZ-763[DBZ-763], https://issues.jboss.org/browse/DBZ-764[DBZ-764]) and Postgres drivers (https://issues.jboss.org/browse/DBZ-912[DBZ-912]).
```

Comment:
Is this sentence correct?

Reply:
You mean grammatically or contents-wise? Can you give the alternative? Thanks!

Reply:
Did not you intend to write "In terms of version upgrades ***we*** migrated"?

Reply:
Ah yes, of course :)